test/test_multiperiod_ac_powerflow.jl

```diff
@@ -119,3 +119,60 @@ end
     solve_powerflow!(data_brute_force; pf = pf)
     @test isnothing(data_brute_force.loss_factors)
 end
+
+@testset failfast = true "MULTI-PERIOD power flows evaluation with DS" for mode in (
```

Comment:
why do you want to `failfast`?

Reply:
I have @test in a for loop for 24 time steps:

```Julia
for time_step in 1:time_steps
        _check_distributed_slack_consistency(
            data.bus_activepower_injection[:, time_step],
            collect(data.bus_slack_participation_factors[:, time_step]),
            init_p_injections[:, time_step],
        )
    end
```

If it fails once, there is no additional benefit for waiting for it to fail more times. and having all the tracebacks in the terminal.